src/model/auto-prompt-config.js

```diff
@@ -127,18 +127,27 @@ export class ImpressionConfig {
 }
 
 /**
- * Predicates of whether or not to show button and prompt.
+ * Predicates to control UI elements.
  */
+// TODO(b/228856600): Move to a more general location.
 export class UiPredicates {
   /**
    * @param {boolean|undefined} canDisplayAutoPrompt
    * @param {boolean|undefined} canDisplayButton
+   * @param {boolean|undefined} purchaseUnavailableRegion
```

Comment:
I think 3 is probably the limit on this but should be okay and not worth changing to object right now. Can you please file another bug (or merge into above) to change to object args though for later?

Reply:
Filed b/229131111.